index.js

```diff
@@ -128,10 +129,34 @@ class TypeWriter extends Component {
   render() {
     const { children, ...props } = this.props;
     const { visibleChars } = this.state;
+    const { fixed } = this.props;
 
+    const visibleString = children.slice(0, visibleChars)
+
+    let components = [(
+      <Text
+        { ...props }
+        key="visible-string">
+        {visibleString}
+      </Text>
+    )];
+    if (fixed) {
+      const invisibleString = children.slice(visibleChars)
+      props.style = {
+        ...props.style,
+        opacity: 0,
+      }
```

Comment:
Instead of mutating `props` here, a new style object should be created and applied to the `<Text />` node:
```jsx
const style = { ...props.style, opacity: 0 }
components.push(
  <Text {...props} style={style} key="invisible-string">{invisibleString}</Text>
)
```